pkg/types/openstack/platform.go

```diff
@@ -43,4 +43,14 @@ type Platform struct {
 	// the default OS image for cluster nodes, or an existing Glance image name.
 	// +optional
 	ClusterOSImage string `json:"clusterOSImage,omitempty"`
+
+	// APIVIP is the static IP on the nodes subnet that the api port for openshift will be assigned
+	// Default: will be set to the 5 on the machinesNetwork CIDR
```

Comment:
`machinesNetwork` -> `machineNetwork`

also it seems like we pick the 5th host in the first network range of the list.